### ISSUE TITLE
Announce selected state to screen readers. 

### ISSUE THREAD

**Issue by blackfalcon** [AlaskaAirlines/auro-pane#32]
## Is your feature request related to a problem? Please describe.

See original issue 

- https://github.com/AlaskaAirlines/auro-pane/issues/12

## Describe the solution you'd like

The scope of this issue is to review the current `selected` functionality within the element and engineer a solution that will leverage that functionality and read back the content of the selected node in the UI. 

## exit criteria 

This issue will be considered closed once the auro-pane element announces the content of the selected node in within the carousel UI.
